app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    @current_user = User.find(1)
+  end
```

Comment:
- [ ] [OPTIONAL] `User.find(1)` will only find a user with an ID of 1, it may give unexpected results if a user is given a different ID. ` You can use `User.first` to find a first user regardless of the ID used.